apps/interactor/command

```diff
@@ -6,10 +6,12 @@ if [[ -z $1 ]]; then
 fi
 
 ARGS=$(printf '{"command": "%s"}' "$1")
-if [[ ! -z $2 ]]; then
+if [[ -n $2 ]]; then
   ARGS=$(printf '{"command": "%s", "args": %s}' "$1" "$2")
 fi
 
-curl -XPUT http://${INTERACTOR_HOST-localhost}:${INTERACTOR_PORT-6100}/v1/lifx/command \
+BIND_ADDRESS=$(lifx --silent --task bind_address)
```

Comment:
so this line I'm gonna veto because the point of the script is not starting photons so it's an instant script.

Reply:
I'm not sure what the purpose of `command` inside the container is anyway. I was just fixing it so it would work, as it didn't on my install, which uses `macvlan` networking for running containers. I guess I could set `INTERACTOR_HOST` _and_ have `host:` in the yaml file.

Reply:
I have reverted this change, but left `curl` in the image so that `command` still works for normal people. 😄

Reply:
yeap, sounds good. I think inside the image command isn't overly useful, but it's fine to leave it